src/index.js

```diff
@@ -1,4 +1,5 @@
 import './styles/index.scss';
+import './../momentum-ui.scss';
```

Comment:
If we're just going to import it like this, doesn't it make sense to move `momentum-ui.scss` into `src`?